test/findTest.js

```diff
@@ -0,0 +1,61 @@
+var db = require('../db')
+var fs = require('fs')
+var async = require('async')
+var collectionName, result
+var done = function(err, resultArray) {result = resultArray}
```

Comment:
these are global parameters and those are evil hellspawn demons from the pit of the antichrist bent of consuming all that is good. move them inside the relevant scope